projects/cm_mcu/Tasks.h

```diff
@@ -30,6 +30,19 @@ void InitTask(void *parameters);
 // ADC task
 #define ADC_CHANNEL_COUNT   21
 #define ADC_INFO_TEMP_ENTRY 20 // this needs to be manually kept correct.
+#ifdef REV1
+#define ADC_INFO_GENFPGA_VCC_INIT_CH 0
+#define ADC_INFO_GENFPGA_VCC_FIN_CH  5
+#define ADC_INFO_FPGA_VCC_INIT_CH    6
+#define ADC_INFO_FPGA_VCC_FIN_CH     7
+#elif defined(REV2) // REV2
+#define ADC_INFO_GENFPGA_VCC_INIT_CH 0
```

Comment:
Maybe `GENFPGA` -> `GEN`? As we discussed everything is FPGA on this board, pretty much, and GENFPGA voltages are also used in clocks, optics, etc.

Reply:
Done.